app/templates/components/forms/admin/content/pages-form.hbs

```diff
@@ -46,7 +45,7 @@
     {{/ui-dropdown}}
   </div>
   <div class="ui buttons">
-    <button class="ui teal button" type="submit">{{t 'Update Page'}}</button>
-    <button class="ui red button">{{t 'Delete Page'}}</button>
+    <button class="ui teal button" type="submit">{{saveButton}}</button>
+    <button class="ui red button">{{cancelButton}}</button>
```

Comment:
@niranjan94 does it means we'll be having three buttons? Save/Update, Cancel, Delete?